### DISCUSSION TITLE
Open Meeting Invite: Code for IBM i Discussion #1

### DISCUSSION THREAD

**Discussion by worksofliam** [codefori/vscode-ibmi#253] [General]
Hey folks. I feel it's time to get together to talk about Code for IBM i! I have a few points I'd like to talk about first and then we can open the floor for anyone's input.

### When is it?

Thursday 26th Aug 2021, 4pm EST / 9pm BST (1 hour - 1.5 hours)
### Where is it?

Zoom: x

**Add to your cal**: 

* [iCal](http://icalgen.yc.sg/?sub=Code+for+IBM+i+Meeting&det=Zoom%3A+https%3A%2F%2Fus02web.zoom.us%2Fj%2F84758120279%3Fpwd%3DYytkbUZ3ZE1za3hRc0hjc1dqT25XQT09&tz=America%2FAnguilla&allday=false&sd=Aug+26%2C+2021&st=4%3A00+PM&ed=Aug+26%2C+2021&et=5%3A30+PM&venue=https%3A%2F%2Fus02web.zoom.us%2Fj%2F84758120279%3Fpwd%3DYytkbUZ3ZE1za3hRc0hjc1dqT25XQT09&isub=true&idt=true&iv=true&dl=1) (also works for Outlook)
* [Google Cal](https://www.google.com/calendar/render?action=TEMPLATE&text=Code%20for%20IBM%20i%20Meeting&dates=20210826T200000Z/20210826T213000Z&details=Code%20for%20IBM%20i%20Meeting%0ADate%20and%20Time%3A%20Aug%2026%2C%202021%208%3A00%20PM%20-%209%3A30%20PM%0AVenue%3A%20https%3A%2F%2Fus02web.zoom.us%2Fj%2F84758120279%3Fpwd%3DYytkbUZ3ZE1za3hRc0hjc1dqT25XQT09%0AZoom%3A%20https%3A%2F%2Fus02web.zoom.us%2Fj%2F84758120279%3Fpwd%3DYytkbUZ3ZE1za3hRc0hjc1dqT25XQT09&location=https%3A%2F%2Fus02web.zoom.us%2Fj%2F84758120279%3Fpwd%3DYytkbUZ3ZE1za3hRc0hjc1dqT25XQT09&trp=true&sf=true&output=xml#f)

### Agenda

* Introductions
* Statistics
   * Installs
   * Reviews
   * GitHub Insights
* Open issues
* Current ideas/PRs

* Main topic: **Creating walkthroughs**
   * What they are
   * How we can use them for the greater good
   * Potential ideas from everyone

* Open floor - let's talk about what matters to our users.

**Comment by szsascha** [comment]
What a great idea!

I'd really like to participate but unfortunately I'm currently on vacation and got a very bad internet connection.

Hopefully there will be some more meetings in the future...

**Comment by RobBoeckermann** [comment]
I just started working with IBM i and this extension has been such a big help! 
I plan on attending to see how I can help contribute to the git extension. I'm hoping to add functionality for branching, checkouts, and merging.

**Reply by worksofliam** [reply]
@RobBoeckermann That's awesome. I will make sure we have time to talk about that.

**Comment by mlstoppa** [comment]
I will attend the discussion, and because I have been using this extension since its first release I hope I will be of help. Thanks Liam

**Reply by worksofliam** [reply]
Excellent! I am glad that you'll be coming.

**Comment by RobBoeckermann** [comment]
@worksofliam does this attached help with any of the debugging information you are trying to get from IBM?
[IBM System i Programming Debugger APIs Version 6 Release 1.pdf](https://github.com/halcyon-tech/vscode-ibmi/files/7062841/IBM.System.i.Programming.Debugger.APIs.Version.6.Release.1.pdf)
"The debugger APIs can be used for program debugging on the system"
